vp/devel/python_peripherals/DLA.py

```diff
@@ -1274,7 +1255,7 @@ def matsum_element_wise(self, A, B):
         if not isinstance(A, list) and not isinstance(B, list):
             return A + B
 
-        assert len(A) == len(B)
+        assert len(A) == len(B), "Assert failed! Different sized operands in matsum"
```

Comment:
Assert style changed with respect to earlier code. Seemed to have parentheses before but not here.

Reply:
Style unified to have parentheses